### ISSUE TITLE
Error compiling music_visualizer example

### ISSUE THREAD

**Issue by targrub** [dtaralla/bevy-rrise#2]
Using code as of https://github.com/dtaralla/bevy-rrise/commit/07c7910fb2cc34f6dbedf4af38dcf4f55653322b

```rust
active toolchain
----------------

stable-x86_64-pc-windows-msvc (default)
rustc 1.65.0 (897e37553 2022-11-02)
```

Running
`cargo run --example music_visualizer --features="bevy/render rrise-headers rrise/AkParametricEQFX"`
gets the result
```rust
error[E0554]: `#![feature]` may not be used on the stable release channel
   --> C:\Users\[xxxx]\.cargo\registry\src\github.com-1ecc6299db9ec823\regex-1.7.0\src\lib.rs:612:42
    |
612 | #![cfg_attr(feature = "pattern", feature(pattern))]
    |                                          ^^^^^^^
```

(The doppler_drone example compiles and runs.)

**Comment by dtaralla** [comment]
Thanks for the report! I'm always using nightly, same for Rrise, so I didn't catch that this one an unstable rust feature. It should be pretty straightforward to replace the unstable usage

**Comment by dtaralla** [comment]
This was actually an issue from the rrise-headers crate.
I've published a new version of rrise-headers (0.2.2) - you can fix this issue on stable 1.61+ toolchains by running `cargo update` then rebuild.
